src/components/styledComponents.js

```diff
@@ -1,10 +1,117 @@
-export const contentWidth = 1200;
+import { createMuiTheme, Button as MuiButton, Typography } from '@material-ui/core';
+import styled from 'styled-components';
+import { styled as muiStyled } from '@material-ui/styles';
+
+export const contentWidth = 1280;
 export const contentWidthPixels = `${contentWidth}px`;
 export const mobileThreshold = 800;
 export const mobileThresholdPixels = `${mobileThreshold}px`;
+export const menuHeightOffset = 71; // 30 + 41 --- 30 for marginTop + 41 menuBar height
+export const theme = createMuiTheme();
```

Comment:
I like the standardization that's going on in this file, but let's try to do as much as we can via [Material UI's theme API](https://material-ui.com/customization/themes/). It has all sorts of stuff for color palette, typography, spacing/layout, etc., and if we customize the theme (then inject it in `layouts/Layout.js` with the `ThemeProvider`), we won't need nearly as much custom CSS and the built-in properties like `variant`, `color`, etc. on the various component APIs will work better.